homeassistant/components/iskra/sensor.py

```diff
@@ -163,6 +167,46 @@ class IskraSensorEntityDescription(SensorEntityDescription):
 )
 
 
+def get_counter_sensor(
```

Comment:
Can we refactor this to return a EntityDescription?

Reply:
for sure, will do